Bio/SeqIO/SnapGeneIO.py

```diff
@@ -62,7 +61,7 @@ def _parse_dna_packet(length, data, record):
         raise ValueError("The file contains more than one DNA packet")
 
     flags, sequence = unpack(">B%ds" % (length - 1), data)
-    record.seq = Seq(sequence.decode("ASCII"), alphabet=Alphabet.generic_dna)
+    record.seq = Seq(sequence.decode("ASCII"))
     record.annotations["molecule_type"] = "DNA"
```

Comment:
Same here.

Reply:
All right. Given the SnapGene format does not seem to be documented, again I suppose it might contain RNA.